application/helpers/admin/statistics_helper.php

```diff
@@ -153,7 +153,7 @@ function createChart($iQuestionID, $iSurveyID, $type, $lbl, $gdata, $grawdata, $
 
             $counter = 0;
             foreach ($lblout as $sLabelName) {
-                $DataSet->SetSerieName(html_entity_decode($sLabelName, null, 'UTF-8'), "Serie" . $counter);
+                $DataSet->SetSerieName(html_entity_decode($sLabelName, ENT_COMPAT, 'UTF-8'), "Serie" . $counter);
```

Comment:
null mean `ENT_COMPAT` ?

Why not 0 ?

Reply:
Not sure what null means here... I incline to 0, and 0 = ENT_HTML401.
Still, after going deeper, I feel we should use `ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401` which is the default for the function and will fit probably the need on this line. Thoughts?

Reply:
> Not sure what null means here... I incline to 0, and 0 = ENT_HTML401.

Then if we need no update : `ENT_HTML401`

>  Still, after going deeper, I feel we should use ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 which is the default for the function and will fit probably the need on this line. Thoughts?

It was my first idea. But null : mean default or 0 ?

Maybe test quickly on 7.4 for the result ?

Reply:
> It was my first idea. But null : mean default or 0 ?

Don't know for sure... to me, it is 0, which matches ENT_COMPAT and ENT_HTML401 as well.

> Maybe test quickly on 7.4 for the result ?

We have been doing some tests about it and feel ` ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401`